cloudbuild.yaml

```diff
@@ -1,33 +1,36 @@
 steps:
+
 - name: 'gcr.io/kaniko-project/executor:latest'
   args:
   - --dockerfile=awscodecommit/Dockerfile
   - --destination=gcr.io/$PROJECT_ID/awscodecommit:latest
   - --cache=true
   waitFor: ['-']
+
 - name: 'gcr.io/kaniko-project/executor:latest'
   args:
   - --dockerfile=awscognito/Dockerfile
   - --destination=gcr.io/$PROJECT_ID/awscognito:latest
   - --cache=true
   waitFor: ['-']
+
 - name: 'gcr.io/kaniko-project/executor:latest'
   args:
   - --dockerfile=awsdynamodb/Dockerfile
   - --destination=gcr.io/$PROJECT_ID/awsdynamodb:latest
   - --cache=true
   waitFor: ['-']
+
 - name: 'gcr.io/kaniko-project/executor:latest'
   args:
   - --dockerfile=awskinesis/Dockerfile
   - --destination=gcr.io/$PROJECT_ID/awskinesis:latest
   - --cache=true
   waitFor: ['-']
+
 - name: 'gcr.io/kaniko-project/executor:latest'
   args:
   - --dockerfile=awssqs/Dockerfile
   - --destination=gcr.io/$PROJECT_ID/awssqs:latest
   - --cache=true
   waitFor: ['-']
-options:
-  machineType: 'N1_HIGHCPU_8'
```

Comment:
@sebgoa this highcpu instance is not required. What was causing the builds to take long to complete was the `microdnf update` command in the `Dockerfile`s which have been removed since we don't need that update.

Reply:
ok, I trust you